R/dettl.R

```diff
@@ -19,13 +19,15 @@
 #' import$transform()
 #' import$load()
 #'
-
+# nolint start
```

Comment:
This seems like a bit of a flaw in lintr that we can't just disable the complexity check here, and that it is flagging up the whole class and not the methods

Reply:
Yeah bit of a shame it doesn't tell us where the actual problem is - we can disable the complexity check for the whole file